core-bundle/tests/Functional/app/config/security.yaml

```diff
@@ -1,14 +1,16 @@
 security:
+    enable_authenticator_manager: true
+
+    password_hashers:
+        Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface: auto
```

Comment:
is this change necessary? Looks like it would force this on all users in the application, not just Contao user objects?

Reply:
We should only apply this to `Contao\User` objects. Fixed in b0669adc4b.